src/Entity/Package.php

```diff
@@ -328,6 +344,31 @@ public function removeReferenceNumber()
     {
         $this->referenceNumber = null;
     }
+    
+    /**
+     * @return ReferenceNumber
+     */
+    public function getReferenceNumber2()
+    {
+        return $this->referenceNumber2;
+    }
+
+    /**
+     * @param ReferenceNumber $referenceNumber
+     *
+     * @return Package
+     */
+    public function setReferenceNumber2(ReferenceNumber $referenceNumber)
+    {
+        $this->referenceNumber2 = $referenceNumber;
+
+        return $this;
+    }
+
+    public function removeReferenceNumber2()
```

Comment:
I don't think this method is really required. It doesn't make a lot of sense to me to first add the second reference number and later remove it again in the same request. Or am I missing something?